documentation/Algorithms_and_Math.md

```diff
@@ -39,7 +39,69 @@ This hybrid approach combines the speed of the secant method with the guaranteed
 ### `Target::interceptPosition`
 This method in the `Target` class formulates the ballistic problem for a *moving* target as a quartic equation (an equation with `t^4` as the highest power) and uses the `Approximate::small_root` solver to find the time-to-intercept (`t`). Once `t` is known, it's straightforward to calculate where the target *will be* at that time and aim there.
 
-## 2. Target Selection and Management
+#### Moving Target Intercept Equations
+The implementation is based on the work of Forrest Smith, as detailed in his blog post "[Solving Ballistic Trajectories](https://www.forrestthewoods.com/blog/solving_ballistic_trajectories/)." The core of the problem is to solve for `t` (time) in the following system of equations, which results in a quartic polynomial.
+
+**Variable Definitions:**
+- `proj_pos`: Projectile initial position vector `(x, y, z)`
+- `target_pos`: Target initial position vector `(x, y, z)`
+- `target_velocity`: Target velocity vector `(vx, vy, vz)`
+- `proj_speed`: Projectile's initial speed (a scalar)
+- `Gv`: Gravity vector `(gx, gy, gz)`
+- `G`: Magnitude of gravity (a scalar)
+
+**Derived Variables (for simplicity):**
+- `diff = target_pos - proj_pos`
+- `H = diff.x`, `J = diff.z`, `K = diff.y`
+- `P = target_velocity.x`, `Q = target_velocity.z`, `R = target_velocity.y`
+- `L = -0.5 * G`
+- `S = proj_speed`
+
+**The Quartic Equation:**
+The problem is reduced to solving `c0*t^4 + c1*t^3 + c2*t^2 + c3*t + c4 = 0`, where the coefficients are:
+
+-   `c0 = L^2`
+-   `c1 = -2 * Q * L`
+-   `c2 = (target_velocity . target_velocity) - 2 * J * L - S^2`
+-   `c3 = 2 * (diff . target_velocity)`
+-   `c4 = diff . diff`
+
+The `Approximate::small_root` function is used to find the smallest, positive, real root of this quartic equation, which represents the time to intercept.
+
+## 2. Mechanical Control and Motor Calculation
+
+### Differential Wrist Mechanism
+
+The physical turret mechanism is a pan-and-tilt system driven by two stationary stepper motors. The motion is mechanically coupled through a set of gears in a configuration analogous to a [differential wrist](https://en.wikipedia.org/wiki/Robotic_wrist#Differential_wrist). This design allows two motors, which are fixed to the base, to control two separate axes of rotation (pan and tilt) at the end of the mechanism.
```

Comment:
This Wikipedia article does not exist.   Please find a new, verifiable, reference.